packages/xorg_libs.rb

```diff
@@ -0,0 +1,272 @@
+require 'package'
+
+class Xorg_libs < Package
+  description 'A collection of xorg libraries.'
+  homepage 'https://www.x.org'
+  version '1.0.0'
+  source_url 'http://downloads.sourceforge.net/project/libtirpc/libtirpc/1.0.2/libtirpc-1.0.2.tar.bz2'
+  source_sha256 '723c5ce92706cbb601a8db09110df1b4b69391643158f20ff587e20e7c5f90f5'
```

Comment:
Why are we using libtirpc source?

Reply:
Fake one. How to use a dummy ?

Reply:
I have needed to do this on several packages myself.  What I usually do is grab the README.md file if the source is from github.com.  What you want is something small related to the project that can be versioned.